File: 1385-find-the-distance-value-between-two-arrays/1385-find-the-distance-value-between-two-arrays.py
class Solution:
    def findTheDistanceValue(self, arr1: List[int], arr2: List[int], d: int) -> int:
        ans = 0
        tmp = 0
        for i in range(len(arr1)):
            for j in range(len(arr2)):
                if abs(arr1[i] - arr2[j]) > d:
                    tmp += 1
            if tmp == len(arr2):
                ans += 1
            tmp = 0
        return ans

File: 1385-find-the-distance-value-between-two-arrays/1385-find-the-distance-value-between-two-arrays.cpp
class Solution {
public:
    int findTheDistanceValue(vector<int>& arr1, vector<int>& arr2, int d) {
        int ans = 0;
        int tmp = 0;
        for (int i=0;i<arr1.size();i++) {
            tmp = 0;
            for (int j=0;j<arr2.size();j++) {
                if (abs(arr1[i]-arr2[j]) > d) tmp += 1;
            }
            if (tmp == arr2.size()) ans += 1;
        }
        return ans;
    }    
};